### PR TITLE
Comm Call - January 2025 - First time contributors

### PR DESCRIPTION
**Pull request by yabellini**
First Draft

### CODE REVIEW

**Comment by yabellini** [comment]
Please, @steffilazerte, can you review it?  
@Bisaloo, we need a short bio for your info.  We have this text from another community call.  Let us know if you want us to use that. 

> Hugo Gruson is an evolutionary biologist who fell in love with R and R package development during his PhD. After this, he moved to a Research Software Engineer job in Montpellier, France, to work full-time on building tools for research in epidemiology.

**Comment by Bisaloo** [comment]
I think the text is mostly still applicable. Maybe something like:

> Hugo Gruson is an evolutionary biologist who fell in love with R and R package development during his PhD. He is now working full-time as an R package developer, with a current focus on making the ecosystem of R packages for epidemiology more robust.

How does this sound?

File: content/commcalls/2025-01-20/index.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    img: /img/community/yaoxiang-li.jpg
+  - name: "Pascal Burkhard"
+    bio: "I'm a geography and computer science teacher in a Swiss high school. 
+    I have been using R for about 15 year now. 
+    I mostly do visualizations for my lessons (graphs and maps). 
+    I'm also a big fan of Quarto to create documents, presentations and books 
+    that can all be organized into clean websites to use as a teaching platform."
+    img: /img/community/pascal-burkhard.png   
+  - name: "Hugo Gruson"
+    bio: ""
```
Comment (yabellini) [inline content/commcalls/2025-01-20/index.md:58]:
```suggestion
    bio: "Hugo Gruson is an evolutionary biologist who fell in love with R and R package development during his PhD. He is now working full-time as an R package developer, with a current focus on making the ecosystem of R packages for epidemiology more robust."
```

File: content/commcalls/2025-01-20/index.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+---
+title: "From Novice to Contributor: Making and Supporting First-Time Contributions to FOSS"
+dateStart: 2025-01-20T16:00:00 # UTC!!
+dateEnd: 2025-01-20T17:00:00 # UTC!!
+date: 2025-01-20T17:00:00 # UTC!! same as dateEnd
+publishDate: 2025-01-20
+description: ""
+author:
+  - Yi-Chin Sunny Tseng
+  - Yaoxiang Li
```
Comment (steffilazerte) [inline content/commcalls/2025-01-20/index.md:10]: We need author pages for Yaoxiang Li and Pascal Burkhard

File: content/commcalls/2025-01-20/index.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    He has made significant contributions to the R ecosystem, 
+    including co-authoring several rOpenSci packages and contributing 
+    to R-core through bug fixes."
+    img: /img/community/yaoxiang-li.jpg
+  - name: "Pascal Burkhard"
+    bio: "I'm a geography and computer science teacher in a Swiss high school. 
+    I have been using R for about 15 year now. 
+    I mostly do visualizations for my lessons (graphs and maps). 
+    I'm also a big fan of Quarto to create documents, presentations and books 
+    that can all be organized into clean websites to use as a teaching platform."
```
Comment (steffilazerte) [inline content/commcalls/2025-01-20/index.md:55]: For consistency with the other bios (third person)
```suggestion
    bio: "Pascal is a geography and computer science teacher in a Swiss high school, and
    has been using R for about 15 years now. 
    Pascal mostly does visualizations for lessons (graphs and maps), but is 
    also a big fan of Quarto to create documents, presentations and books 
    that can all be organized into clean websites to use as a teaching platform."
```

File: content/commcalls/2025-01-20/index.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+
+Contributing to open source can be very rewarding, but also incredibly intimidating. 
+When we asked about first time contributions on the rOpenSci Slack, 
+people recalled the challenges and mistakes they made, 
+but mostly how much they learned from and enjoyed that experience.
+
+In this community call moderated by Hugo Gruson, our speaker, 
+Yaoxiang Li will share Best Practices and Advanced Techniques for Supercharging 
+R Package Quality with testthat, Pascal Burkhard will discuss is the basic git
```
Comment (steffilazerte) [inline content/commcalls/2025-01-20/index.md:77]: I think we need a bit of "first-time contribution" context
```suggestion
Yaoxiang Li will discus the importance of tests with respect to first-time contributions, and share best practices and advanced techniques for supercharging 
R package quality with testthat, Pascal Burkhard will discuss is the basic git 
```

**Review by steffilazerte — COMMENTED** [review]
Hi @yabellini 

Sorry it took me so long to get to this.

Two comments and a couple of wording suggestions, other than that, it's good!

- [ ] We need author files for Yaoxiang Li and Pascal Burkhard
- [ ] Is this the order the panelists will speak in? I wonder if we should move Yaoxiang Li a bit later, as his topic is a bit more specific, and start with either Sunny or Pascal? Who's topics are a bit more general? Or is Hugo giving a bit of an overview? 
- [ ] Oh and I think the order of authors should be the same in the bios, the text and the authors list (you know how picky I am 😅 )

**Pull request merged**